src/containers/ProjectMonitoring.js

```diff
@@ -0,0 +1,5518 @@
+import React from 'react'
+import Flexbox from 'flexbox-react'
+import { connect } from 'react-redux'
+import GraphHeartbeats from '../components/GraphHeartbeats'
+import GraphHttpRequests from '../components/GraphHttpRequests'
+import GraphOsData from '../components/GraphOsData'
+import GraphUrls from '../components/GraphUrls'
+import OsDataWidget from '../components/OsDataWidget'
+import HttpRequestWidget from '../components/HttpRequestWidget'
+
+const mapStateToProps = state => ({
+  session: state.session,
+  projects: state.projects,
+  projectMembers: state.projectMembers
+})
+
+const mapDispatchToProps = {
+}
+
+
+const mockData = {
```

Comment:
Could you move the test data to separate file and import it from there? Just create a separate folder called test data or something. It will make it easier to work with the component rather than having to scroll through a 6000 line component.